logging/logs.rb

```diff
@@ -0,0 +1,19 @@
+#Types of logs:
+#   log.debug "a very nice little debug message"
+#   log.info "things are operating nominally"
+#   log.warn "this is your last warning"
+#   log.error StandardError.new("something went horribly wrong")
+#   log.fatal "I Die!"
+
+require 'logging'
+require 'environment.rb'
+
+# Creates a logfile with the level 'debug'
+#   that appends to log_file.log
+log = Logging.logger['log_file']
```

Comment:
I think having the logging setup in a separate file is not a bad idea, especially if we want to make it more sophisticated in the future. Environment would be fine too, but I don't see any reason to change it now. We definitely want to use an env variable for the log file location, though, which I think is implemented now.